src/ducks/transfers/TransferPage.spec.jsx

```diff
@@ -59,3 +83,57 @@ describe('transfer page', () => {
     expect(errorView.props().error).toBe(err)
   })
 })
+
+describe('personal info', () => {
+  const setup = ({ myselfData } = {}) => {
+    const client = createMockClient({
+      queries: {
+        [recipientsConn.as]: {
+          doctype: recipientsConn.query().doctype,
+          data: recipients
+        },
+        [accountsConn.as]: {
+          doctype: accountsConn.query().doctype,
+          data: accounts
+        },
+        [myselfConn.as]: {
+          doctype: myselfConn.query().doctype,
+          data: [
+            {
+              _id: 'myselfid1234',
+              ...myselfData
+            }
+          ]
+        }
+      }
+    })
+    const root = render(
+      <AppLike client={client}>
+        <TransferPage />
+      </AppLike>
+    )
+
+    return { root }
+  }
+
+  afterEach(() => {
+    flag('banks.transfers.need-personal-information', false)
+  })
+
+  it('should show personal info if myself contact not sufficiently filled', async () => {
+    window.innerWidth = 300 // isMobile -> true
```

Comment:
Out of curiosity, why the explicit test on mobile?

Reply:
Because otherwise on desktop, the transfer stepper is shown behind the modal. I've added a test for desktop that checks that the stepper is shown behind the modal.